### ISSUE TITLE
can't use inline style properties

### ISSUE THREAD

**Issue by badaverse** [wiredmax/react-flags#8]
I wanna use vertical-align style in flag.
but this component ignore inline style properties.
do you have any plans to apply this functionality?

**Comment by badaverse** [comment]
and className also.

**Comment by wiredmax** [comment]
No, not at the moment, I'm quite busy at work currently, but fell free to sumit me a pull request.

**Comment by cescoferraro** [comment]
@wiredmax webpack-dev-server is missing from the dependencies.
Which version are you using?

**Comment by ivanuy** [comment]
> @wiredmax webpack-dev-server is missing from the dependencies.
> Which version are you using?

@cescoferraro  @wiredmax this PR might be helpful to just add webpack-dev-server to dev dependencies. Please check this out [Add webpack-dev-server as devDependency](https://github.com/wiredmax/react-flags/pull/15)
